layouts/ucf-post-list-card.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * The default functions for the card layout
+ **/
+if ( ! function_exists( 'ucf_post_list_display_card_before' ) ) {
+	function ucf_post_list_display_card_before( $posts, $list_title ) {
+		ob_start();
+	?>
+		<div class="ucf-post-list card-layout">
+	<?php
+		echo ob_get_clean();
+	}
+
+	add_action( 'ucf_post_list_display_card_before', 'ucf_post_list_display_card_before', 10, 2 );
+}
+
+
+if ( !function_exists( 'ucf_post_list_display_card_title' ) ) {
+
+	function ucf_post_list_display_card_title( $posts, $list_title ) {
+
+		if ( $list_title ) {
+			$formatted_title = '<h2 class="ucf-post-list-title">' . $list_title . '</h2>';
+		}
+
+		echo $formatted_title;
+	}
+
+	add_action( 'ucf_post_list_display_card_title', 'ucf_post_list_display_card_title', 10, 2 );
+
+}
+
+if ( ! function_exists( 'ucf_post_list_display_card' ) ) {
+	function ucf_post_list_display_card( $posts, $show_image, $posts_per_row, $list_title ) {
+		if ( ! is_array( $posts ) ) { $posts = array( $posts ); }
```

Comment:
In this plugin, $posts can be false here--be sure to check that $posts is not false before wrapping it in an array. See https://github.com/UCF/UCF-Post-List-Shortcode/pull/3/files#diff-49707eb69856160844c43c14940d90e3R35